Settings/Structure/Default.json

```diff
@@ -0,0 +1 @@
+{"stick_mode": false, "bonds": true, "ball_size": 0.9999999999999992, "stick_size": 1.0}
```

Comment:
@Michel-Heinz That's an interesting value, but I guess it just has to be smaller than the stick size.

Reply:
Ah nice, I did not see that. I think this is just double precision magic